.github/workflows/build_starter.yml

```diff
@@ -28,87 +28,94 @@ on:
+      additional_cmake_flags: ${{ github.event.inputs.additional_cmake_flags }}
+  
+  build_ios:
+    name: build-ios-unity${{ github.event.inputs.unity_version }}-CPP${{ github.event.inputs.firebase_cpp_sdk_version }}
+    uses: ./.github/workflows/build_ios.yml
+    needs: check_and_prepare
+    if: contains(needs.check_and_prepare.outputs.platform, 'iOS')
+    with:
+      unity_version: ${{ github.event.inputs.unity_version }}
+      firebase_cpp_sdk_version: ${{ github.event.inputs.firebase_cpp_sdk_version }}
+      unity_branch: ${{ github.event.inputs.unity_branch }}
+      apis: ${{ github.event.inputs.apis }}
+      unity_platform_name: iOS
+      additional_cmake_flags: ${{ github.event.inputs.additional_cmake_flags }}
 
-      - name: Build macos
-        uses:  ./.github/workflows/build_macos.yml
-        with:
-          unity_version: ${{ github.event.inputs.unity_branch }}
-          firebase_cpp_sdk_version: ${{ github.event.inputs.firebase_cpp_sdk_version }}
-          unity_branch: ${{ github.event.inputs.unity_branch }}
-          apis: ${{ github.event.inputs.apis }}
-          unity_platform_name: macOS,iOS
-          additional_cmake_flags: ${{ github.event.inputs.additional_cmake_flags }}
+  build_linux:
+    name: build-linux-unity${{ github.event.inputs.unity_version }}-CPP${{ github.event.inputs.firebase_cpp_sdk_version }}
+    uses: ./.github/workflows/build_linux.yml
+    needs: check_and_prepare
+    if: contains(needs.check_and_prepare.outputs.platform, 'Linux')
+    with:
+      unity_version: ${{ github.event.inputs.unity_version }}
+      firebase_cpp_sdk_version: ${{ github.event.inputs.firebase_cpp_sdk_version }}
+      unity_branch: ${{ github.event.inputs.unity_branch }}
+      apis: ${{ github.event.inputs.apis }}
+      unity_platform_name: Linux
+      additional_cmake_flags: ${{ github.event.inputs.additional_cmake_flags }}
 
-      - name: Build windows
-        uses:  ./.github/workflows/build_windows.yml
-        with:
-          unity_version: ${{ github.event.inputs.unity_branch }}
-          firebase_cpp_sdk_version: ${{ github.event.inputs.firebase_cpp_sdk_version }}
-          unity_branch: ${{ github.event.inputs.unity_branch }}
-          apis: ${{ github.event.inputs.apis }}
-          unity_platform_name: Windows
-          additional_cmake_flags: ${{ github.event.inputs.additional_cmake_flags }}
+  build_macos:
+    name: build-macos-unity${{ github.event.inputs.unity_version }}-CPP${{ github.event.inputs.firebase_cpp_sdk_version }}
+    uses: ./.github/workflows/build_macos.yml
+    needs: check_and_prepare
+    if: contains(needs.check_and_prepare.outputs.platform, 'macOS')
+    with:
+      unity_version: ${{ github.event.inputs.unity_version }}
+      firebase_cpp_sdk_version: ${{ github.event.inputs.firebase_cpp_sdk_version }}
+      unity_branch: ${{ github.event.inputs.unity_branch }}
+      apis: ${{ github.event.inputs.apis }}
+      unity_platform_name: macOS,iOS
```

Comment:
iOS?

Reply:
I'm mapping this from the existing sdk build process, which does need both of the component installed.